rpcs3/CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ else()
 	set(ADDITIONAL_LIBS "")
 endif()
 
+if(UNIX)
```

Comment:
@hcorion I'm not saying that these shouldn't be left in (I used the word "abandoned", not "dropped"). I'm just explaining the current state of OSX support. Besides, I'm not sure it'll be easy to integrate/add dependency on a commercial project for open source rpcs3 (would it work in CI?). But that's not relevant to this PR.